tests/unit_tests/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "unit_tests",
+  "version": "1.0.0",
+  "description": "JSR unit tests.",
+  "main": "",
+  "scripts": {
+    "test": "./start_unit_tests.sh",
+    "start": "./start_unit_tests.sh"
+  },
+  "author": "Martin Bednar, ibednar@fit.vutbr.cz",
+  "license": "GNU General Public License",
```

Comment:
The license is missing the version. I suggest "3 or later" to be compatible with the rest of the project

Reply:
Thanks for your advice. I was not sure about the license. I have resolve this suggestion in the commit #[2bf11cd2770572099735767379db471253c59a66](https://github.com/polcak/jsrestrictor/pull/84/commits/2bf11cd2770572099735767379db471253c59a66).